rmgpy/data/kinetics/family.py

```diff
@@ -47,12 +48,14 @@
 from rmgpy import settings
 from rmgpy.reaction import Reaction
 from rmgpy.kinetics import Arrhenius
+from rmgpy.kinetics.arrhenius import ArrheniusBM
```

Comment:
For consistency, could you add `ArrheniusBM` to `kinetics.__init__` and combine this import with the prior one for `Arrhenius`?

Reply:
added in atg4